index.html

```diff
@@ -4,7 +4,182 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=BIZ+UDPGothic:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/home.css" />
     <title>Geniune charity and care</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="top-header">
+          <ul class="top-header-navbar">
+            <li class="nav-item">
+              <a href="#" class="nav-link"
+                ><i class="fa-brands fa-facebook-f"></i
+              ></a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="nav-item"><a href="#" class="nav-link">English</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+          </ul>
+        </div>
+        <div class="header-bottom">
+          <div class="logo"><a href="#">My Logo</a></div>
+          <div class="nav-right">
+            <ul class="main-nav">
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campain">GC Campain</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="wrapper">
+      <section class="introduction">
+        <span class="quote">
+          “In helping others, we shall help ourselves, for whatever good we give
+          out completes the circle and comes back to us.”
+        </span>
+        <h1 class="goal">
+          Practicing the better exercice of the heart by reaching down and
+          lifting people up.
+        </h1>
+        <div class="ideal">
+          <span
+            >Love is not what we say, love is what we do. As a song is not a
+            song until we sing it. A bell is not a bell until we ring it. In the
+            same way <strong>Love</strong> is not love until we show it
+            away</span
+          >
+          <h3 class="work-hour">Everyday 8am - 5pm(MON) ~ (FRI) 2pm</h3>
+          @ Genuine charity, from all over the places
+        </div>
```

Comment:
- [ ] **Required:** To have more resemblance to the template, implement the below:

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/163602400-279dc249-8c9a-4256-a6aa-a78f57c19eb2.png) | ![image](https://user-images.githubusercontent.com/27709832/163602674-cadd570d-5ee2-44b5-beff-1e0451cb83bb.png)


- Increase the h1 size and add the background image as the template.
- Use the correct font type for the header labelled 1 on the image above.
- Increase the size of the element labelled 2 on the image above.

Reply:
The font type is the same as you can see the name `cocogoose` and `Lato` and that what I have used. :100:

Reply:
resolved 👍